src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -151,10 +157,25 @@ object SeaCreatureTracker {
         tracker.renderDisplay(config.position)
     }
 
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        val data = event.getConstant<ItemsJson>("Items")
+        armorNames.clear()
+        armorNames.addAll(data.trophy_armors)
+    }
+
     fun resetCommand(args: Array<String>) {
         tracker.resetCommand(args, "shresetseacreaturetracker")
     }
 
     private fun isEnabled() = LorenzUtils.inSkyBlock &&
-        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled
+        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled && !isTrophyFishing()
+
+    private fun isTrophyFishing(): Boolean {
+        val boots = InventoryUtils.getBoots()?.getInternalName().toString().replace("internalName:", "")
+        val leggings = InventoryUtils.getLeggings()?.getInternalName().toString().replace("internalName:", "")
+        val chestplate = InventoryUtils.getChestplate()?.getInternalName().toString().replace("internalName:", "")
+        val helmet = InventoryUtils.getHelmet()?.getInternalName().toString().replace("internalName:", "")
```

Comment:
Use .asString instead to remove this weirdness

Reply:
makes sense something like this would exist